monitors/ContractMonitor.js

```diff
@@ -48,6 +49,52 @@ class ContractMonitor {
     return eventArray[eventArray.length - 1].blockNumber;
   }
 
+  // Quries NewSponsor events since the latest query marked by `lastNewSponsorBlockNumber`.
+  checkForNewSponsors = async () => {
+    this.logger.debug({
+      at: "ContractMonitor",
+      message: "Checking for new sponsor events",
+      lastNewSponsorBlockNumber: this.lastNewSponsorBlockNumber
+    });
+
+    // Get the latest new sponsor information.
+    let latestNewSponsorEvents = this.empEventClient.getAllNewSponsorEvents();
+
+    // Get events that are newer than the last block number we've seen
+    let newSponsorEvents = latestNewSponsorEvents.filter(event => event.blockNumber > this.lastNewSponsorBlockNumber);
+
+    for (let event of newSponsorEvents) {
+      // Check if new sponsor is UMA bot.
+      const isLiquidatorBot = this.monitoredLiquidators.indexOf(event.sponsor);
+      const isDisputerBot = this.monitoredDisputers.indexOf(event.sponsor);
+      const isUMABot = Boolean(isLiquidatorBot != -1 || isDisputerBot != -1);
```

Comment:
the logic you've got here is good but ideally let's not use the `UMA` name in the source code. This could be `isMonitoredBot` perhaps?

Reply:
good idea

Reply:
+1